### PR TITLE
Enhancement for generic playbooks with Cybereason.

### PR DESCRIPTION
**Pull request by altmannyarden**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/23014

## Description
New sub-playbooks for Isolate endpoint generic and Search endpoints by hash with Cybereason.

## Minimum version of Demisto
3.5.0

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Playbooks/playbook-Isolate_Endpoint_-_Cybereason.yml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#default#':
+      - '4'
+      yes:
+      - '3'
+    separatecontext: false
+    conditions:
+    - label: yes
```
Comment (yaakovi) [inline Playbooks/playbook-Isolate_Endpoint_-_Cybereason.yml:94]:
```suggestion
    - label: 'yes'
```

File: Playbooks/playbook-Isolate_Endpoint_-_Cybereason.yml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#default#':
+      - '4'
+      yes:
+      - '2'
+    separatecontext: false
+    conditions:
+    - label: yes
```
Comment (yaakovi) [inline Playbooks/playbook-Isolate_Endpoint_-_Cybereason.yml:45]:
```suggestion
    - label: 'yes'
```

File: Playbooks/playbook-Isolate_Endpoint_-_Cybereason.yml [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+      version: -1
+      name: Is Cybereason integration enabled?
+      description: "Check if Cybereason active in the system.\n"
+      type: condition
+      iscommand: false
+      brand: ''
+    nexttasks:
+      '#default#':
+      - '4'
+      yes:
```
Comment (yaakovi) [inline Playbooks/playbook-Isolate_Endpoint_-_Cybereason.yml:41]:
```suggestion
      'yes':
```

**Comment by altmannyarden** [comment]
@yaron-libman Please review the new playbooks 
Packs/Legacy/Playbooks/playbook-Search_Endpoints_By_Hash_-_Cybereason.yml
Packs/Legacy/Playbooks/playbook-Isolate_Endpoint_-_Cybereason.yml

**Comment by yaron-libman** [comment]
@altmannyarden Please add an initial description for the *playbook-Isolate_Endpoint-_Cybereason.yml* file.

**Comment by altmannyarden** [comment]
@yaron-libman The system API (Cybereason) supports only MD5 and SHA1.
So if a user will insert sha256 or other hash, he will get an error.
image

**Comment by yaron-libman** [comment]
@altmannyarden From what I can see, there's still no description for *playbook-Isolate_Endpoint-_Cybereason.yml*. Please fix that so i can approve the PR.

**Comment by yaron-libman** [comment]
> @yaron-libman The system API (Cybereason) supports only MD5 and SHA1.
> So if a user will insert sha256 or other hash, he will get an error.
> image

Cool - thanks

**Pull request merged**